site/docs/v1/tech/apis/_user-response-body.adoc

```diff
@@ -35,6 +35,9 @@ True if the User is active. False if the User has been deactivated. Deactivated
 [field]#user.birthDate# [type]#[String]#::
 The User's birthdate formatted as `YYYY-MM-DD`
 
+[field]#user.connectorId# [type]#[UUID]#::
+The connector associated with the data source used to authenticate the user.  The link:/docs/v1/tech/connectors/fusionauth-connector[FusionAuth Connector] is the default.
```

Comment:
Did you review this for terminology? https://fusionauth.io/docs/v1/tech/connectors/
I think we have thus far primarily used "system of record" instead of data source. 

You don't need to document defaults in the response body, only in the request.

Reply:
I just hadn't gotten to this yet.  Pushing an update.